frontend/src/Author/Delete/DeleteAuthorModalContent.js

```diff
@@ -125,7 +125,7 @@ class DeleteAuthorModalContent extends Component {
             deleteFiles &&
               <div className={styles.deleteFilesMessage}>
                 <div>
-                  {translate('TheAuthorFolderStrongpathstrongAndAllOfItsContentWillBeDeleted')}
+                  {translate('TheAuthorFolderAndAllOfItsContentWillBeDeleted', [path])}
```

Comment:
@ta264  does this Fix #1191  ?

Reply:
If the translation has a {0} in it, yes it will